inc/class-rest-api.php

```diff
@@ -405,31 +405,50 @@ protected function save_term_meta( int $term_id, $request ): bool {
 			$term_id,
 			$request
 		);
+		$nested_meta = apply_filters(
+			'sst_pre_save_term_meta_nested',
+			$request['nestedMeta'] ?? [],
+			$term_id,
+			$request
+		);
 
-		if ( empty( $meta ) ) {
+		if ( empty( $meta ) && empty( $nested_meta ) ) {
 			return false;
 		}
 
-		foreach ( $meta as $key => $values ) {
-			// Discern between single values and multiple.
-			if ( is_array( $values ) ) {
-				delete_term_meta( $term_id, $key );
-				foreach ( $values as $value ) {
-					add_term_meta(
+		if ( ! empty( $meta ) ) {
+			foreach ( $meta as $key => $values ) {
+				// Discern between single values and multiple.
+				if ( is_array( $values ) ) {
+					delete_term_meta( $term_id, $key );
+					foreach ( $values as $value ) {
+						add_term_meta(
+							$term_id,
+							$key,
+							$this->replace_refs_in_meta_value( $value, $key )
+						);
+					}
+				} else {
+					update_term_meta(
 						$term_id,
 						$key,
-						$this->replace_refs_in_meta_value( $value, $key )
+						$this->replace_refs_in_meta_value( $values, $key )
 					);
 				}
-			} else {
-				update_term_meta(
-					$term_id,
-					$key,
-					$this->replace_refs_in_meta_value( $values, $key )
-				);
 			}
 		}
 
+		if ( ! empty( $nested_meta ) ) {
+			if ( ! empty( $nested_meta ) ) {
```

Comment:
Looks like this conditional is unnecessary (repeats the previous line). I think I did the same above on line 374 (not part of this PR, but if you want to fix!).